src/coreComponents/physicsSolvers/multiphysics/poromechanicsKernels/PoromechanicsKernels.cmake

```diff
@@ -113,7 +113,20 @@ endif( )
 
 set( kernelNames ThermoPoromechanicsKernels )
 set( subregionList CellElementSubRegion )
-set( porousSolidDispatch PorousSolid<ElasticIsotropic> )
+set( porousSolidDispatch PorousSolid<DruckerPragerExtended>
```

Comment:
if we dispatch on all of them you can just add ThermoPoromechanicsKernels to the first list.

Reply:
Like in set( kernelNames PoromechanicsKernels 
                                         ThermoPoromechanicsKernels )

Reply:
You mean also removing `set( porousSolidDispatch PorousSolid<DruckerPragerExtended>` and following porousSolidDispatch? But, locally, compilation error was reported:

/usr/bin/ld: lib/libphysicsSolvers.so: undefined reference to `double geos::thermalPoromechanicsKernels::ThermalMultiphasePoromechanics<geos::CellElementSubRegion, geos::constitutive::PorousSolid<geos::constitutive::ModifiedCamClay>, geos::finiteElement::ConformingVirtualElementOrder1<14, 7> >::kernelLaunch<RAJA::PolicyBaseT<(RAJA::Policy)3, (RAJA::Pattern)1, (RAJA::Launch)0, (camp::resources::v1::Platform)1, RAJA::policy::omp::Parallel, RAJA::wrapper<RAJA::policy::omp::omp_for_schedule_exec<RAJA::policy::omp::Auto> > >, geos::thermalPoromechanicsKernels::ThermalMultiphasePoromechanics<geos::CellElementSubRegion, geos::constitutive::PorousSolid<geos::constitutive::ModifiedCamClay>, geos::finiteElement::ConformingVirtualElementOrder1<14, 7> > >(int, geos::thermalPoromechanicsKernels::ThermalMultiphasePoromechanics<geos::CellElementSubRegion, geos::constitutive::PorousSolid<geos::constitutive::ModifiedCamClay>, geos::finiteElement::ConformingVirtualElementOrder1<14, 7> > const&)'

Reply:
I should have fixed it.